numpy/core/_add_newdocs.py

```diff
@@ -5148,7 +5154,7 @@
     out : ndarray, None, or tuple of ndarray and None, optional
         A location into which the result is stored. If not provided or `None`,
         a freshly-allocated array is returned. For consistency with
-        :ref:`ufunc.__call__`, if given as a keyword, this may be wrapped in a
+        ``ufunc.__call__``, if given as a keyword, this may be wrapped in a
```

Comment:
`ufunc.__call__` has no docstring. Should it?

Reply:
I think `__call__` is always an implementation detail, whatever it does must be described elsewhere (e.g. in the main docstring of the object).